020_app_L44/020_app_L44_unsafe.py

```
@@ -0,0 +1,108 @@
+def definite_rce(payload: str) -> None:
+    subprocess.run(payload, shell=True)
+
+
+@app.route("/rce/<string:payload>")
+def potential_rce_1(payload: str) -> None:
+    subprocess.run(["echo", payload])
+
+
+@app.route("/rce/<int:payload>")
+def potential_rce_2(payload: int) -> None:
+    subprocess.run(f"echo {payload}", shell=True)
```

Ruff (0.14.1)

`subprocess` call with `shell=True` identified, security issue

(S602)